OxfordBodleian/Juel-Jensen/BDLaethg41.xml

```diff
@@ -50,35 +152,176 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                </dimensions>
+                            </decoNote>
+                            
+                            <decoNote xml:id="d3" type="miniature" corresp="#pic3">
+                                <desc></desc>
+                                <dimensions unit="mm">
+                                    <height>132</height>
+                                    <width>70–74</width>
+                                </dimensions>
+                            </decoNote>
+                            
+                            <decoNote xml:id="d4" type="miniature" corresp="#pic4">
+                                <desc></desc>
+                                <dimensions unit="mm">
+                                    <height>110</height>
+                                    <width>73</width>
+                                </dimensions>
+                            </decoNote>
                         </decoDesc>-->
                         
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                   <desc>The names of the owners, mentioned in supplications throughout, were 
+                                       <persName ref="PRS13856Elyas"/>, <persName ref="PRS13857Baselyos"/>, <persName ref="PRS13858MaryamS"/>,
+                                       and <persName ref="PRS13859Nesebbeyo"/>.
+                                   Spaces for the name or the owner(s) had been left empty by the scribe and were filled subsequently. In the upper
+                                   part of the scroll, the words introducing the supplications (<foreign xml:lang="gez">ነፍሳ</foreign>,
+                                       <foreign xml:lang="gez">ሥጋሃ</foreign>, <foreign xml:lang="gez">አድኅና</foreign>)
+                                       are used with their feminine form, which was
+                                   then corrected to the masculine form.
+                                   </desc>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <desc>The manuscript contains no scribal corrections other than the adaptation of the supplications
+                                        to the gender of the owner.</desc>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <desc>The verso is empty.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1">The two parchment strips are sewn together with narrow parchment strips.</decoNote>
```

Comment:
We could add some more information, without going too much in detail, saying 'the two strips are slightly overlapped and sewn together with a running stitch using two narrow parchment strips'.